package.json

```diff
@@ -12,11 +12,12 @@
   },
   "scripts": {
     "build-all": "npm-run-posix-or-windows build-all:task",
-    "build-all:task": "(yarn build-cdt-lib & yarn build-extension & yarn build-devtools & yarn build-lr & yarn build-viewer & wait) && yarn build-pack",
-    "build-all:task:windows": "yarn build-cdt-lib && yarn build-extension && yarn build-devtools && yarn build-lr && yarn build-viewer",
+    "build-all:task": "(yarn build-cdt-lib & yarn build-extension & yarn build-devtools & yarn build-lr & yarn build-viewer & yarn build-firehouse & wait) && yarn build-pack",
+    "build-all:task:windows": "yarn build-cdt-lib && yarn build-extension && yarn build-devtools && yarn build-lr && yarn build-viewer && yarn build-firehouse",
     "build-cdt-lib": "node ./build/build-cdt-lib.js",
     "build-extension": "node ./build/build-extension.js",
     "build-devtools": "yarn link && yarn link lighthouse && node ./build/build-bundle.js clients/devtools-entry.js dist/lighthouse-dt-bundle.js && node ./build/build-dt-report-resources.js",
+    "build-firehouse": "node ./build/build-firehouse.js",
```

Comment:
can we rename firehouse to something else? :)
i forget what it is pretty much everytime i read it.  sorry.

i'm hoping we can have something that makes sense within https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-cli/test/smokehouse/readme.md#pipeline (which also will need an update in this PR)

does `frontends/bundled.js` or `frontends/browserified.js` work?    
`build-bundled-smokehouse` i suppose?

Reply:
`frontend/bundled` would conflate with `lighthouse-runners/bundle.js`.

what about `frontend/lib` (library b/c it is meant to be used + configured)? then we have a yarn script for `build-smokehouse-lib` that generates a `smokehouse-bundle.js`.

Reply:
that sounds great

Reply:
@paulirish done